storybook/src/admin/form/Select.tsx

```diff
@@ -62,18 +61,6 @@ function Story() {
                                         )}
                                     </Field>
 
-                                    <Field name="flavorClearable" label="Clearable Flavor" fullWidth>
```

Comment:
Do we have a story to show an optional select field?

Reply:
Any select field that is not required shows this behaviour. (Minus the multiple select field, see my [question](https://github.com/vivid-planet/comet/pull/2294#issuecomment-2245246215) above)